src/Common/CustomerPortalClient.cs

```diff
@@ -73,16 +73,18 @@ public async Task<Transport> GetMessageBusTransport()
                 _session.LogDebug($"Configuring message bus...");
 
                 // create new transport using the config
-                string transportConfigString = (await new GetApplicationBootInformationInput().GetApplicationBootInformationAsync(true)).TransportConfig;
+                var session = (await new GetApplicationBootInformationInput().GetApplicationBootInformationAsync(true));
+                string transportConfigString = session.TransportConfig;
                 TransportConfig transportConfig = JsonConvert.DeserializeObject<TransportConfig>(transportConfigString);
                 transportConfig.ApplicationName = "Customer Portal Client";
                 transportConfig.TenantName = ClientConfigurationProvider.ClientConfiguration.ClientTenantName;
+                transportConfig.SecurityToken = transportConfig.SecurityToken == null ? _session.AccessToken : transportConfig.SecurityToken;

```

Comment:
I could be mistaken, but I thought that it only worked with the refresh token (or PAT) and not the Access Token. I say this because I remembered the change that you made to not retrieve the refresh token anymore in favor of retrieving only the access token and that didn't work

Reply:
What happened was with the token / update pat I couldn't connect to the Message Bus. Also, previously, the token was named as accessToken and I switched to the token because it's not the access token actually.